CMakeLists.txt

```diff
@@ -9,9 +9,17 @@ project(
     LANGUAGES CXX
 )
 
-include(CTest)
+if(${CMAKE_SOURCE_DIR} STREQUAL ${CMAKE_BINARY_DIR})
+    message(FATAL_ERROR "In-source builds are not allowed!")
+endif()
+
+include(CPack)
```

Comment:
Are we using cpack for anything yet?

Reply:
just to be prepared to create a binary distribution.